.github/workflows/tests_fe.yml

```diff
@@ -0,0 +1,26 @@
+name: Testing FE
+
+on: pull_request
+
+jobs:
+  build_and_test:
+    name: Testing # job name (unique id)
```

Comment:
Since we test FE it's better to mention it in the name like `Frontend testing`

Reply:
For now I named it 'Testing FE', because backend tests in our git actions is named 'Testing BE'